app/src/main/java/com/android/feedme/ml/Barcode.kt

```diff
@@ -50,21 +50,20 @@ fun barcodeScanner(
  * callbacks for both successful and failed outcomes.
  *
  * @param barcodeNB The barcode number for which the product name needs to be extracted.
- * @param onSuccess Callback function invoked when the product name extraction is successful. It
- *   takes a single parameter, [productName], representing the extracted product name.
+ * @param onSuccess Callback function invoked when the product name extraction is successful.
  * @param onFailure Callback function invoked when an exception occurs during the extraction
- *   process. It takes a single parameter, [exception], representing the exception that occurred.
+ *   process.
  * @throws Exception if an error occurs during the extraction process.
  */
 suspend fun extractProductNameFromBarcode(
```

Comment:
In this function in terms of good pratice just keep the exception as an input for the onFailure function, even if for now it is useless. We might need it down the line.

Reply:
Got it !